pkg/configuration/kubefed.go

```diff
@@ -0,0 +1,58 @@
+package configuration
```

Comment:
why do you use a separate file? It's a bit confusing having methods on the `Registry` type in another file

Reply:
I thought it would be better to have the kubefed configuration separate and it seemed like a logical way to divide the configuration into pieces but not if it makes it more confusing. There isn't a ton of configuration right now so I moved it back and we can always revisit later if it does grow too large.